commerce/views/order.py

```diff
@@ -43,10 +43,18 @@ def dispatch(self, request, *args, **kwargs):
 
         if success:
             messages.success(request, message)
-            return redirect(commerce_settings.SUCCESSFUL_PAYMENT_REDIRECT_URL)
+
+            if commerce_settings.SUCCESSFUL_PAYMENT_REDIRECT_URL:
```

Comment:
SUCCESSFUL_PAYMENT_REDIRECT_URL setting can not be empty. Why do we need this condition exactly? Default value is '/'. If we need something else, let's change it in the project settings.